samanthas_telegram_bot/main.py

```diff
@@ -12,56 +15,26 @@
     filters,
 )
 
-from samanthas_telegram_bot.conversation.callbacks.common import (
-    cancel,
-    check_if_review_needed_give_review_menu_or_ask_final_comment,
-    redirect_to_coordinator_if_registered_check_chat_id_ask_first_name,
-    review_requested_item,
-    say_bye_if_does_not_want_to_register_another_or_ask_first_name,
-    send_help,
-    start,
-    store_additional_help_comment_ask_final_comment,
-    store_age_ask_timezone,
-    store_comment_end_conversation,
-    store_data_ask_another_level_or_communication_language_or_start_assessment,
-    store_email_check_existence_ask_role,
-    store_first_name_ask_last_name,
-    store_last_name_ask_source,
-    store_locale_ask_if_already_registered,
-    store_non_teaching_help_ask_another_or_additional_help,
-    store_one_time_slot_ask_another,
-    store_phone_ask_email,
-    store_role_ask_age,
-    store_source_check_username,
-    store_teaching_language_ask_another_or_level_or_communication_language,
-    store_timezone_ask_slots_for_one_day_or_teaching_language,
-    store_username_if_available_ask_phone_or_email,
+import samanthas_telegram_bot.conversation.callbacks.registration.common as common
+import samanthas_telegram_bot.conversation.callbacks.registration.student as student
+import samanthas_telegram_bot.conversation.callbacks.registration.teacher as teacher
+from samanthas_telegram_bot.api_queries.auxil.enums import LoggingLevel
+from samanthas_telegram_bot.auxil.log_and_notify import log_and_notify
+from samanthas_telegram_bot.data_structures.context_types import (
+    CUSTOM_CONTEXT_TYPES,
+    BotData,
+    ChatData,
+    UserData,
 )
-from samanthas_telegram_bot.conversation.callbacks.student import (
-    ask_communication_language_after_smalltalk,
-    ask_communication_language_or_start_assessment_depending_on_learning_experience,
-    assessment_store_answer_ask_question,
-    send_smalltalk_url_or_ask_communication_language,
-    store_communication_language_ask_non_teaching_help_or_start_review,
-)
-from samanthas_telegram_bot.conversation.callbacks.teacher import (
-    store_communication_language_ask_teaching_experience,
-    store_experience_ask_about_groups_or_speaking_clubs,
-    store_frequency_ask_student_age_groups,
-    store_number_of_groups_ask_frequency,
-    store_peer_help_ask_another_or_additional_help,
-    store_readiness_to_host_speaking_clubs_ask_additional_help_or_bye,
-    store_student_age_group_ask_another_or_non_teaching_help,
-    store_teachers_additional_skills_ask_if_review_needed,
-    store_teaching_preference_ask_groups_or_frequency_or_student_age,
-)
-from samanthas_telegram_bot.conversation.constants_enums import ConversationState as State
-from samanthas_telegram_bot.conversation.user_data import UserData
+from samanthas_telegram_bot.data_structures.enums import ConversationState as State
+
+load_dotenv()
 
 logging.basicConfig(
     format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
```

Comment:
Now you don't have option to see your debug logs. Maybe put logging level to env vars?

Reply:
Very nice idea. Done